gadget/install/install.go

```diff
@@ -51,9 +52,24 @@ func deviceFromRole(lv *gadget.LaidOutVolume, role string) (device string, err e
 	return "", fmt.Errorf("cannot find role %s in gadget", role)
 }
 
+func roleOrLabelOrNode(part gadget.OnDiskStructure) string {
+	switch {
+	case part.Role != "":
+		return part.Role
+	case part.Label != "":
+		return part.Label
+	case part.Node != "":
+		return part.Node
+	case part.Name != "":
```

Comment:
anyway given how it is used, and my other comment, I would leave out Node and make this:

roleOrLabelOrName